src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentGeneratorControllerIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.fprl;
+
+import io.restassured.response.Response;
+import net.serenitybdd.rest.SerenityRest;
+import org.apache.http.HttpStatus;
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.TestPropertySource;
+
+
+@SpringBootTest(classes = DocumentGeneratorControllerIntegrationTest.class)
+@AutoConfigureMockMvc
+@TestPropertySource("/application-local.properties")
```

Comment:
rename property source file , to point to application.properties

Reply:
Property file reference changed